### PR TITLE
Added pagination on reporters. Get program memberships/groups types and  users  that manage program

### PR DESCRIPTION
**Pull request by CR1AT0RS**
Added pagination on reporters. Added attributes on reporters to choose page_size or get a page_number. 

example uage:
program.reporters(page_number: 1, page_size: 25)

### CODE REVIEW

**Comment by esjee** [comment]
Cool! The code looks good to me, but could you add a test?

**Comment by oreoshake** [comment]
It would be cool if there were an autopagination option like octokit has (maybe it’s implemented in a gem or something). These seem like good defaults to start with so we can add autopagination later.

**Comment by CR1AT0RS** [comment]
Added: Get all users that manage the program. Program groups and membership type. Updated documentation. For auto pagination people can just a add for loop in their code.

File: lib/hackerone/client/program.rb [hunk 1]
```diff
@@ -24,6 +24,12 @@ def id
       def attributes
         OpenStruct.new(@program[:attributes])
       end
+       # List all groups, membership types and users for a program
+      def users()
```
Comment (oreoshake) [inline lib/hackerone/client/program.rb]: I wonder if it's better to just expose the private `members` method instead.

**Pull request closed without merging**